OutOfSchool/OutOfSchool.BusinessLogic/Services/Database/WorkshopService.cs

```diff
@@ -352,7 +359,9 @@ async Task<Workshop> UpdateWorkshopLocally()
             dto.AddressId = currentWorkshop.AddressId;
             dto.Address.Id = currentWorkshop.AddressId;
 
-            await ChangeTeachers(currentWorkshop, dto.Teachers ?? new List<TeacherDTO>()).ConfigureAwait(false);
+            await ChangeTeachers(currentWorkshop, dto.Teachers ?? []).ConfigureAwait(false);
+            
+            contactsService.ProcessUpdate(currentWorkshop, dto);
```

Comment:
Shouldn't it be part of `ChangeTeachers`

Reply:
Still undone - was it missed or was there any reasons?

Reply:
@VadymLevkovskyi Teachers are not related to contacts, as we're providing contacts of the workshop itself, not its teachers. Also later teachers will be completely changed to new `employees`, so instead of adding teachers as value objects they are going to be completely separate and indepentent entity.